includes/transformer/class-post.php

```diff
@@ -642,9 +642,16 @@ protected function get_type() {
 			return \ucfirst( $post_format_setting );
 		}
 
-		$has_title = post_type_supports( $this->wp_object->post_type, 'title' );
-
-		if ( ! $has_title ) {
+		$has_title = \post_type_supports( $this->wp_object->post_type, 'title' );
+		$content   = \get_post_field( 'post_content', $this->wp_object->ID );
+		$content   = \wp_strip_all_tags( $content );
+
+		// Check if the post has a title.
+		if (
+			! $has_title ||
+			! $this->wp_object->post_title ||
+			\strlen( $content ) <= ACTIVITYPUB_NOTE_LENGTH
+		) {
```

Comment:
Would it make sense to add a check for whether the post has the `aside` post format to determine whether it's a note? (not sure what constitutes a note)

Reply:
We do that already: https://github.com/Automattic/wordpress-activitypub/blob/00a57c39a4e844949bc4f64a45714d1f21d9645d/includes/transformer/class-post.php#L667

`Article` is only used for the `Standard` Post-Format, every other will be "translated" to `Note`. This current change is only for cases, where users don't use custom-post-formats and every post defaults to `Article`.